types/snowflake-sdk/index.d.ts

```diff
@@ -163,6 +163,21 @@ export interface SnowflakeError extends SnowflakeErrorExternal {
     externalize?: () => SnowflakeErrorExternal;
 }
 
+export enum validFetchAsStringType {
+    STRING = 'STRING',
+    BOOLEAN = 'BOOLEAN',
+    NUMBER = 'NUMBER',
+    DATE = 'DATE',
+    JSON = 'JSON',
+}
+
+// The supported types are: String, Boolean, Number, Date, and JSON.
+export interface StatementOptions {
```

Comment:
Maybe we should call this stream options since that's how it's referenced [here](https://github.com/snowflakedb/snowflake-connector-nodejs/blob/4d009ee97c031b3bbf55ecdc8fbce81b41d86dc9/lib/connection/result/row_stream.js#L32).
```suggestion
export interface StreamOptions {
```

Reply:
Thanks, Will update them. Thanks for helping me point this.